package NYT.NExecNode.NProto;

import "yt_proto/yt/client/chunk_client/proto/chunk_spec.proto";
import "yt/ytlib/chunk_client/proto/data_source.proto";

////////////////////////////////////////////////////////////////////////////////

message TArtifactKey
{
    optional bytes format = 3;
    repeated NYT.NChunkClient.NProto.TChunkSpec chunk_specs = 6;
    required NYT.NChunkClient.NProto.TDataSource data_source = 7;
    optional int32 access_method = 8 [default = 1]; // ELayerAccessMethod
    optional int32 filesystem = 9 [default = 1]; // ELayerFilesystem
    optional string nbd_device_id = 10;
}

////////////////////////////////////////////////////////////////////////////////
